### PR TITLE
Unified routing

### LINKED ISSUE #300: Unify routing naming
image

We should have one, a consistent prefix for all ShopAPI related routes.

### PR DESCRIPTION
**Pull request by mamazu**
Fixes #300

### CODE REVIEW

File: src/Resources/config/routing/register.yml [hunk 1]
```diff
@@ -1,28 +1,28 @@
-sylius_shop_api_register:
+shop_api_register:
```
Comment (lchrusciel) [inline src/Resources/config/routing/register.yml]: Can we move back to `sylius_shop_api` prefix, and change rest of routes to follow same pattern as requested by @pamil in https://github.com/Sylius/ShopApiPlugin/issues/300#issuecomment-417588413

Comment (mamazu) [reply src/Resources/config/routing/register.yml]: No problem.

**Comment by mamazu** [comment]
Done

**Comment by lchrusciel** [comment]
Thanks @mamazu! 🐋

**Pull request merged**